tests/services/inventory/test_data/additional_cai_resources.dump

```diff
@@ -60,8 +60,22 @@
+# Kubernetes Node (id 920x)
+{"name":"//container.googleapis.com/projects/1042/zones/us-central1-b/clusters/test-cluster/k8s/namespaces/kube-system/pods/9201","asset_type":"k8s.io/Pod","resource":{"version":"v1","discovery_document_uri":"https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json","discovery_name":"io.k8s.api.core.v1.Pod","data":{"id":"9201","metadata":{"annotations":{"scheduler.alpha.kubernetes.io/critical-pod":"","seccomp.security.alpha.kubernetes.io/pod":"docker/default"},"creationTimestamp":"2019-05-01T01:47:13Z","generateName":"test-kube-","labels":{"k8s-app":"test-label,"pod-template-hash":"123"},"name":"test-kube-k798l","namespace":"kube-system","ownerReferences":[{"apiVersion":"apps/v1","blockOwnerDeletion":true,"controller":true,"kind":"ReplicaSet","name":"test-kube","uid":"123"}],"resourceVersion":"100332589","selfLink":"/api/v1/namespaces/kube-system/pods/test-kube","uid":"9201"},"spec":{"containers":[{"command":["/cluster-proportional-autoscaler","--namespace=kube-system","--configmap=test-configmap","--target=Deployment/kube-dns","--default-params={\"linear\":{\"coresPerReplica\":256,\"nodesPerReplica\":16,\"preventSinglePointFailure\":true}}","--logtostderr=true","--v=2"],"image":"k8s.gcr.io/image","imagePullPolicy":"IfNotPresent","name":"autoscaler","resources":{"requests":{"cpu":"20m","memory":"10Mi"}},"terminationMessagePath":"/dev/termination-log","terminationMessagePolicy":"File","volumeMounts":[{"mountPath":"/var/run/secrets/kubernetes.io/serviceaccount","name":"test-kube","readOnly":true}]}],"dnsPolicy":"ClusterFirst","nodeName":"test-node","priority":2000000000,"priorityClassName":"system-cluster-critical","restartPolicy":"Always","schedulerName":"default-scheduler","securityContext":{},"serviceAccount":"test-sa”,"serviceAccountName":"test-sa","terminationGracePeriodSeconds":30,"tolerations":[{"key":"CriticalAddonsOnly","operator":"Exists"},{"effect":"NoExecute","key":"node.kubernetes.io/not-ready","operator":"Exists","tolerationSeconds":300},{"effect":"NoExecute","key":"node.kubernetes.io/unreachable","operator":"Exists","tolerationSeconds":300}],"volumes":[{"name":"test-volume","secret":{"defaultMode":420,"secretName":"test"}}]},"status":{"conditions":[{"lastProbeTime":null,"lastTransitionTime":"2019-05-01T01:47:13Z","status":"True","type":"Initialized"},{"lastProbeTime":null,"lastTransitionTime":"2019-05-01T01:47:16Z","status":"True","type":"Ready"},{"lastProbeTime":null,"lastTransitionTime":"2019-05-01T01:47:13Z","status":"True","type":"PodScheduled"}],"containerStatuses":[{"containerID":"docker://123","image":"k8s.gcr.io/cluster-proportional-autoscaler-amd64:1.1.2-r2","imageID":"docker://sha256:123","lastState":{},"name":"autoscaler","ready":true,"restartCount":0,"state":{"running":{"startedAt":"2019-05-01T01:47:15Z"}}}],"hostIP":"10.128.0.8","phase":"Running","podIP":"10.60.0.45","startTime":"2019-05-01T01:47:13Z"}}}}
```

Comment:
I think it's worth reimporting the sample data as these have changed substantially from what is really returned from the API.

The project you should be using for the test clusters is project1: https://github.com/forseti-security/forseti-security/blob/dev/tests/services/inventory/test_data/mock_gcp_results.py#L5983

So for example the API returned this for Namespace:

{"name":"//container.googleapis.com/projects/cicd-prod/zones/us-central1-b/clusters/redditmobile-canary-central/k8s/namespaces/default","asset_type":"k8s.io/Namespace","resource":{"version":"v1","discovery_document_uri":"https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json","discovery_name":"io.k8s.api.core.v1.Namespace","data":{"metadata":{"creationTimestamp":"2016-11-12T18:00:39Z","name":"default","resourceVersion":"6","selfLink":"/api/v1/namespaces/default","uid":"eb8a1c53-a901-11e6-8876-42010af00195"},"spec":{"finalizers":["kubernetes"]},"status":{"phase":"Active"}}}}

The test data should look like this:

{"name":"//container.googleapis.com/projects/project1/zones/us-central1-b/clusters/test-cluster/k8s/namespaces/default","asset_type":"k8s.io/Namespace","resource":{"version":"v1","discovery_document_uri":"https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json","discovery_name":"io.k8s.api.core.v1.Namespace","data":{"metadata":{"creationTimestamp":"2016-11-12T18:00:39Z","name":"default","resourceVersion":"6","selfLink":"/api/v1/namespaces/default","uid":"9221"},"spec":{"finalizers":["kubernetes"]},"status":{"phase":"Active"}}}}

I changed the following fields **(in bold)**:

{"name":"//container.googleapis.com/projects/**project1**/zones/us-central1-b/clusters/**test-cluster**/k8s/namespaces/default","asset_type":"k8s.io/Namespace","resource":{"version":"v1","discovery_document_uri":"https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json","discovery_name":"io.k8s.api.core.v1.Namespace","data":{"metadata":{"creationTimestamp":"2016-11-12T18:00:39Z","name":"default","resourceVersion":"6","selfLink":"/api/v1/namespaces/default","uid":"**9221**"},"spec":{"finalizers":["kubernetes"]},"status":{"phase":"Active"}}}}

Every resource for the test should use:

**project1** for the projectId
**test-cluster** for the cluster name
**default** for the namespace
And **uid** value should be changed to the unique ID for that resource type, like **9221**

Beyond that, you should replace any identifying information, but not change the format of the data.

Reply:
Done